common/static/coffee/spec/discussion/discussion_spec_helper.coffee

```diff
@@ -37,560 +37,34 @@ class @DiscussionSpecHelper
         )
 
     @setUnderscoreFixtures = ->
-        for templateName in ['thread-show']
+        templateNames = [
+            'thread', 'thread-show', 'thread-edit',
+            'thread-response', 'thread-response-show', 'thread-response-edit',
+            'response-comment-show', 'response-comment-edit',
+            'thread-list-item', 'discussion-home', 'search-alert',
+            'new-post', 'thread-type', 'new-post-menu-entry',
+            'new-post-menu-category', 'topic', 'post-user-display',
+        ]
+        templateNamesNoTrailingTemplate = [
+            'forum-action-endorse', 'forum-action-answer', 'forum-action-follow',
+            'forum-action-vote', 'forum-action-report', 'forum-action-pin',
+            'forum-action-close', 'forum-action-edit', 'forum-action-delete',
+            'forum-actions',
+        ]
+
+        for templateName in templateNames
             templateFixture = readFixtures('common/templates/discussion/' + templateName + '.underscore')
             appendSetFixtures($('<script>', { id: templateName + '-template', type: 'text/template' })
                 .text(templateFixture))
+        for templateName in templateNamesNoTrailingTemplate
+            templateFixture = readFixtures('common/templates/discussion/' + templateName + '.underscore')
```

Comment:
I just noticed that these files live under lms/ not common/.   yet the tests work; do you understand why?  if so could you add a comment explaining why, to avoid confusion?